Api/Presentation/ExpensesController.cs

```diff
@@ -1,59 +1,108 @@
 using Api.Domain;
-
+using Api.Domain.Services;
 using AutoMapper;
-
 using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
 
 [ApiController]
 [Route("api/expenses")]
 public class ExpensesController : ControllerBase
 {
     private readonly IMapper _mapper;
+    private readonly IExpenseService _expenseService;
 
-    public ExpensesController(IMapper mapper)
+    public ExpensesController(IMapper mapper, IExpenseService expenseService)
     {
         _mapper = mapper;
+        _expenseService = expenseService;
+    }
+
+    [HttpPost]
+    public async Task<IActionResult> CreateExpense([FromBody] CreateExpenseRequest createExpenseRequest)
```

Comment:
Add the default value "CreatedAt" attribute in the Expense model, and please wait a bit to use the user id ([Authentication and Autorization PR](https://github.com/programacion-6/ExpenseTrackerGrupo5/pull/8)), otherwise there may be empty or invalid fields.